include/sparrow/layout/null_array.hpp

```diff
@@ -183,12 +199,63 @@ namespace sparrow
      * null_array implementation *
      *****************************/
 
+    inline null_array::null_array(
+        size_t length,
+        std::optional<std::string_view>&& name,
+        std::optional<std::string_view>&& metadata
```

Comment:
These arguments should be passed by value so that the user can decide to copy or move.

Reply:
done